lib/org/chromium/webidl/CanonicalizerRunner.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'CanonicalizerRunner',
+  extends: 'org.chromium.webidl.PipelineRunner',
+
+  requires: ['org.chromium.webidl.Canonicalizer'],
+
+  constants: {
+    MISSING_PROPS: 'Missing required properties in PipelineMessage',
```

Comment:
ditto from other PRs: Elaborate message with specifics of runner class and what's missing. No need  for a constant at this time.